frontend/src/features/documents/documentUpload/DocumentUploadContainer.tsx

```diff
@@ -48,10 +61,11 @@ export const DocumentUploadContainer: React.FunctionComponent<IDocumentUploadCon
     <DocumentUploadView
       documentTypes={documentTypes}
       isLoading={retrieveDocumentMetadataLoading || uploadDocumentLoading}
-      mayanMetadata={[]}
+      mayanMetadata={documentTypeMetadata}
```

Comment:
the variable names is a tad confusing. What do you think about 
mayanMetadata -> mayanDocumentTypeMetadata
documentMetadata -> mayanDocumentMetadata

Reply:
removed